beacon-chain/rpc/beacon/assignments.go

```diff
@@ -14,6 +14,8 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+const epochError = "Cannot retrieve information about an epoch in the future, current epoch %d, requesting %d"
```

Comment:
Can you please change all `...Error` to `err...`? E.g. `errEpoch`. This is the convention we use. Also, this name should be more expressive: `errNoEpochInfo`, just like in `beacon-chain/rpc/beacon/validators_test.go`.